doc/classes/Tween.xml

```diff
@@ -219,8 +219,14 @@
 			<return type="Tween" />
 			<param index="0" name="ease" type="int" enum="Tween.EaseType" />
 			<description>
-				Sets the default ease type for [PropertyTweener]s and [MethodTweener]s animated by this [Tween].
-				If not specified, the default value is [constant EASE_IN_OUT].
+				Sets the default ease type for [PropertyTweener]s and [MethodTweener]s appended after this method.
+				Before this is set, the default ease type is [constant EASE_IN_OUT].
+				[codeblock]
+				var tween := create_tween()
+				tween.tween_property(self, "position", Vector2(300, 0), 0.5) # Will use ease type EASE_IN_OUT
+				tween.set_ease(Tween.EASE_IN)
+				tween.tween_property(self, "rotation_degrees", 45.0, 0.5) # Will use ease type EASE_IN
```

Comment:
I would suggest keeping the comments short for readability, as there's a lot of values to read through. With syntax highlighting it's not that bad now, but still.
```suggestion
				var tween := create_tween()
				tween.tween_property(self, "position", Vector2(300, 0), 0.5) # Uses EASE_IN_OUT.
				tween.set_ease(Tween.EASE_IN)
				tween.tween_property(self, "rotation_degrees", 45.0, 0.5) # Uses EASE_IN.
				
```